### PR TITLE
add Go binding for addressplan and addressspaceplan resources

### PR DESCRIPTION
**Pull request by k-wall**
Adds Go bindings for addressplan and addressspaceplan resources (admin.enmasse.io, v1beta2). Unit tests added.

### CODE REVIEW

File: pkg/apis/admin/v1beta2/doc.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright 2018, EnMasse authors.
```
Comment (lulf) [inline pkg/apis/admin/v1beta2/doc.go]: Update year (applies to other files as well)

File: pkg/apis/admin/v1beta2/types.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	Spec AddressPlanSpec `json:"spec"`
+}
+
+type AddressPlanSpec struct {
+	AddressType      string             `json:"addressType"`
+	DisplayName      string             `json:"displayName,omitempty"`
+	LongDescription  string             `json:"longDescription,omitempty"`
+	ShortDescription string             `json:"shortDescription,omitempty"`
+	DisplayOrder     int                `json:"displayOrder,omitempty"`
+	Resources        map[string]float32 `json:"resources,omitempty"`
```
Comment (lulf) [inline pkg/apis/admin/v1beta2/types.go]: Since its a Double in java, I think this should be float64.

File: pkg/apis/admin/v1beta2/types.go [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+type AddressSpacePlanSpec struct {
+	AddressPlans     []string           `json:"addressPlans"`
+	AddressSpaceType string             `json:"addressSpaceType"`
+	DisplayName      string             `json:"displayName,omitempty"`
+	LongDescription  string             `json:"longDescription,omitempty"`
+	ShortDescription string             `json:"shortDescription,omitempty"`
+	InfraConfigRef   string             `json:"infraConfigRef,omitempty"`
+	DisplayOrder     int                `json:"displayOrder,omitempty"`
+	ResourceLimits   map[string]float32 `json:"resourceLimits,omitempty"`
```
Comment (lulf) [inline pkg/apis/admin/v1beta2/types.go]: float32 -> float64

File: pkg/apis/admin/v1beta2/doc.go (hunk 1 above)
Comment (k-wall) [reply pkg/apis/admin/v1beta2/doc.go]: Fixed.

File: pkg/apis/admin/v1beta2/types.go (hunk 2 above)
Comment (k-wall) [reply pkg/apis/admin/v1beta2/types.go]: Fixed.

File: pkg/apis/admin/v1beta2/types.go (hunk 3 above)
Comment (k-wall) [reply pkg/apis/admin/v1beta2/types.go]: Fixed.

**Pull request merged**